src/daemon/daemon.cpp

```diff
@@ -3173,6 +3206,46 @@ mp::MountHandler::UPtr mp::Daemon::make_mount(VirtualMachine* vm, const std::str
                : vm->make_native_mount_handler(config->ssh_key_provider.get(), target, mount);
 }
 
+void mp::Daemon::configure_new_interfaces(const std::string& name, mp::VirtualMachine& vm, mp::VMSpecs& specs)
+{
+    std::vector<size_t> new_interfaces;
+
+    for (auto i = 0u; i < specs.extra_interfaces.size(); ++i)
```

Comment:
This is a typical range iteration, right? Could this be a range-for instead?

Reply:
The index i is used inside the loop, you can use range-for, but you still need to declare variable i and increaments it, which does not make it cleaner.

Reply:
Oh OK, I missed that. Thanks for clarifying!

Reply:
Hmmm, I need the index of each element. I think the clearest way of doing it is as it is now. WDYT?